### ISSUE TITLE
Mist Multisig on 2 Separate Machines No Longer Working

### ISSUE THREAD

**Issue by taoteh1221** [ethereum/mist#805]
Node type: geth
OS: Ubuntu 14.04 LTS
Network type: main
Mist version: 0.7.4

I have Mist 0.7.4 on 2 separate Machines, both running Ubuntu 14.04 LTS. I have a 4-of-4 Multisig Wallet, with 2 accounts on one machine, and 2 accounts on the other machine. 

I went and synced up both machine's blockchains, and then proceeded to initiate a withdrawal from the multisig wallet using Mist, for an amount was over the daily limit set on it. In the past (a few months ago) I was able to do this and get a "Approve / Reject" prompt on both machines via Mist. Unfortunately this time I only got the "Approve / Reject" prompt on the machine I initiated the transaction on. The other machine gave me no "Approve / Reject" prompt at all. Both machines were fully in sync with the blochchain at all times, receiving new blocks on a regular basis in the Mist status area at the top of the GUI. 

The only way to resolve this issue was for me to temporarily copy the other 2 account files over to the machine that initiated the transaction, which significantly reduced my level of security if that machine ever had a keylogger trojan running on it.

**Comment by luclu** [comment]
@taoteh1221 could you please try to replicate this with the latest [release 0.7.5](https://github.com/ethereum/mist/releases)?

**Comment by taoteh1221** [comment]
@luclu Will do. I am installing an SSD drive in my desktop PC Friday, I already put one in the laptop the other week and bumped it's RAM up to 8GB...so I will test v0.7.5 Friday after I get the desktop PC upgraded as well, to minimize any possible latency issues. Looks like you all have been VERY busy, looking forward to taking v0.7.5 for a test drive. :+1:

**Comment by taoteh1221** [comment]
@luclu So I tried v0.7.5 on both machines today to see if I could replicate this multisig GUI bug still, after swapping my HDD for a SSD (so both machines now have SSDs). My 8 core desktop loved v0.7.5 and it ran great, BUT my laptop consistently froze every single time after chain sync when the Mist application opens. I retried a few times, between system reboots. I even let it sit for a couple hours to see what happens. RAM / CPU usage was minimal after 2 hours, BUT it was still frozen AND eventually the entire GUI window went to a blank white. Other than this Mist issue, my system and other apps on it are 100% stable. 

Here are the CPU specs, in case you happen to have this hardware yourself to test with (important note: this is a 4 core celeron with integrated video that requires <= kernel 3.16 due to a kernel video bug introduced in higher versions...if you attempt to replicate issues on this hardware, you'll need the v3.16 kernel or lower...I am running 8GB RAM on this machine):
==============================================================================
processor       : 0
vendor_id       : GenuineIntel
cpu family      : 6
model           : 55
model name      : Intel(R) Celeron(R) CPU  N2940  @ 1.83GHz
stepping        : 8
microcode       : 0x829
cpu MHz         : 2700.242
cache size      : 1024 KB
physical id     : 0
siblings        : 4
core id         : 0
cpu cores       : 4
apicid          : 0
initial apicid  : 0
fpu             : yes
fpu_exception   : yes
cpuid level     : 11
wp              : yes
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse 
sse2 ss ht tm pbe syscall nx rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc aperfmperf
 pni pclmulqdq dtes64 monitor ds_cpl vmx est tm2 ssse3 cx16 xtpr pdcm sse4_1 sse4_2 movbe popcnt tsc_deadline_timer rdra
nd lahf_lm 3dnowprefetch ida arat epb dtherm tpr_shadow vnmi flexpriority ept vpid tsc_adjust smep erms
bogomips        : 3660.80
clflush size    : 64
cache_alignment : 64
address sizes   : 36 bits physical, 48 bits virtual
power management:
==============================================================================

So I guess testing the multisig is on hold for now. Let me know if you'd like me to open a seperate issue for this new bug, and I'll create it for you all to remember it easier. Thanks for all your hard work.

**Comment by taoteh1221** [comment]
I tried v0.7.6 on the machine that gave a blank white window with v0.7.5, and that side issue was gone. Yay. :-)  So I tried doing 4-of-4 multisig transactions on v0.7.6 over daily limits with 2 accounts per machine, with a wallet contract upgrade as well (for the recent security vulnerability discovered), and everything finally worked fairly well this time. 

My slower machine's possible latency issues did cause the same "Approve / Reject" prompt not showing issue one time, BUT after a restart of Mist I was successfully able to initiate an over-daily-limit transaction from either machine, and the opposite machine responded in kind with it's own "Approve / Reject" prompt. It did seem that initiating these 4-of-4 multisig transactions on the slower machine first was more reliable...maybe a latency issue with my slower hardware keeping up with the remotely-called state changes from the other machine being scanned well by Mist I guess.

The only real issue I noticed was when the last account signature finished fully approving the transaction, the "Approve / Reject" prompt did NOT disappear, nor did the final account icon get replaced with a green checkmark...but the transaction did go through fine. Upon restart of Mist, the "Approve / Reject" prompt disappeared as it should.

Thanks for all your VERY hard work, I'll mark this issue as closed. Let me know if you want me to open a separate issue for the prompt not disappearing.
